code/analyticsedge/src/test/java/com/adobe/marketing/mobile/FakeSystemInfoService.java

```diff
@@ -0,0 +1,48 @@
+/*
+ Copyright 2021 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile;
+
```

Comment:
Instead of creating a class can we mock the behavior using Mockito?

Reply:
I switched to using a class based on the discussion in slack. We can discuss more about it so we can decide what we want to do going forward.

Reply:
Sure, we decided not to use PowerMockito, For this case Mockito should be sufficient.

Reply:
i see, i'll use Mockito